api/backups/restore.go

```diff
@@ -23,6 +28,15 @@ var (
 	}
 )
 
+// isUpgradeInProgressErr returns whether or not the error
+// is an "upgrade in progress" error.  This is necessary as
+// the error type returned from a facade call is rpc.RequestError
+// and we cannot use params.IsCodeUpgradeInProgress
+func isUpgradeInProgressErr(err error) bool {
```

Comment:
The err.Code from this error is empty, and the message actually is inUpgradeError from apiserver/upgrading_root.go.  I used the method the bootstrap command used to inspect the error.